commands/video.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
         name: 'add',
         description: `Takes you through a CLI flow to add a ${featureName} resource to your local backend`,
       },
+      {
+        name: 'version',
```

Comment:
Should this be alphabetical?

Reply:
sure, i'll alphabetize this and amplify-plugins.json